`print_verbose` makes build fail when `scu_build=yes`

### Tested versions

- 4.4.beta

### System information

Windows 11

### Issue description

The `print_verbose` macro is causing one of the CI jobs that uses `scu_build=yes` to fail in my PR although I didn't modify the file calling it.
https://github.com/godotengine/godot/actions/runs/12934119444/job/36074414239

My local build also fails when I use `scu_build=yes` but works fine otherwise.
```sh
[100%] godot\core/debugger/remote_debugger_peer.cpp(172): error C3861: 'print_verbose': identifier not found
```

By the way, it works if I replace it with `WARN_VERBOSE`.

### Steps to reproduce

- Checkout https://github.com/godotengine/godot/pull/100516
- I changed calls to `print_verbose`  in `core/debugger/remote_debugger_peer.cpp` to `WARN_VERBOSE` to fix CI. Change it back to `print_verbose`. 
- Build it with `scons scu_build=yes`.

### Minimal reproduction project (MRP)

N/A

--- a/core/variant/variant_utility.cpp
+++ b/core/variant/variant_utility.cpp
@@ -999,9 +999,7 @@ void VariantUtilityFunctions::print_rich(const Variant **p_args, int p_arg_count
 	r_error.error = Callable::CallError::CALL_OK;
 }
 
-#undef print_verbose
-
-void VariantUtilityFunctions::print_verbose(const Variant **p_args, int p_arg_count, Callable::CallError &r_error) {
+void VariantUtilityFunctions::_print_verbose(const Variant **p_args, int p_arg_count, Callable::CallError &r_error) {
 	if (OS::get_singleton()->is_stdout_verbose()) {
 		String s;
 		for (int i = 0; i < p_arg_count; i++) {
@@ -1581,16 +1579,16 @@ static _FORCE_INLINE_ Variant::Type get_ret_type_helper(void (*p_func)(P...)) {
 	};                                                                                                           \
 	register_utility_function<Func_##m_func>(#m_func, m_args)
 
-#define FUNCBINDVARARGV(m_func, m_args, m_category)                                                              \
+#define FUNCBINDVARARGV_CNAME(m_func, m_func_cname, m_args, m_category)                                          \
 	class Func_##m_func {                                                                                        \
 	public:                                                                                                      \
 		static void call(Variant *r_ret, const Variant **p_args, int p_argcount, Callable::CallError &r_error) { \
 			r_error.error = Callable::CallError::CALL_OK;                                                        \
-			VariantUtilityFunctions::m_func(p_args, p_argcount, r_error);                                        \
+			VariantUtilityFunctions::m_func_cname(p_args, p_argcount, r_error);                                  \
 		}                                                                                                        \
 		static void validated_call(Variant *r_ret, const Variant **p_args, int p_argcount) {                     \
 			Callable::CallError c;                                                                               \
-			VariantUtilityFunctions::m_func(p_args, p_argcount, c);                                              \
+			VariantUtilityFunctions::m_func_cname(p_args, p_argcount, c);                                        \
 		}                                                                                                        \
 		static void ptrcall(void *ret, const void **p_args, int p_argcount) {                                    \
 			Vector<Variant> args;                                                                                \
@@ -1624,6 +1622,8 @@ static _FORCE_INLINE_ Variant::Type get_ret_type_helper(void (*p_func)(P...)) {
 		}                                                                                                        \
 	};                                                                                                           \
 	register_utility_function<Func_##m_func>(#m_func, m_args)
+
+#define FUNCBINDVARARGV(m_func, m_args, m_category) FUNCBINDVARARGV_CNAME(m_func, m_func, m_args, m_category)
 
 #define FUNCBIND(m_func, m_args, m_category)                                                                     \
 	class Func_##m_func {                                                                                        \
@@ -1832,7 +1832,7 @@ void Variant::_register_variant_utility_functions() {
 	FUNCBINDVARARGV(printt, sarray(), Variant::UTILITY_FUNC_TYPE_GENERAL);
 	FUNCBINDVARARGV(prints, sarray(), Variant::UTILITY_FUNC_TYPE_GENERAL);
 	FUNCBINDVARARGV(printraw, sarray(), Variant::UTILITY_FUNC_TYPE_GENERAL);
-	FUNCBINDVARARGV(print_verbose, sarray(), Variant::UTILITY_FUNC_TYPE_GENERAL);
+	FUNCBINDVARARGV_CNAME(print_verbose, _print_verbose, sarray(), Variant::UTILITY_FUNC_TYPE_GENERAL);
 	FUNCBINDVARARGV(push_error, sarray(), Variant::UTILITY_FUNC_TYPE_GENERAL);
 	FUNCBINDVARARGV(push_warning, sarray(), Variant::UTILITY_FUNC_TYPE_GENERAL);
 

--- a/core/variant/variant_utility.h
+++ b/core/variant/variant_utility.h
@@ -133,8 +133,7 @@ struct VariantUtilityFunctions {
 	static String type_string(Variant::Type p_type);
 	static void print(const Variant **p_args, int p_arg_count, Callable::CallError &r_error);
 	static void print_rich(const Variant **p_args, int p_arg_count, Callable::CallError &r_error);
-#undef print_verbose
-	static void print_verbose(const Variant **p_args, int p_arg_count, Callable::CallError &r_error);
+	static void _print_verbose(const Variant **p_args, int p_arg_count, Callable::CallError &r_error);
 	static void printerr(const Variant **p_args, int p_arg_count, Callable::CallError &r_error);
 	static void printt(const Variant **p_args, int p_arg_count, Callable::CallError &r_error);
 	static void prints(const Variant **p_args, int p_arg_count, Callable::CallError &r_error);

